src/ov_utils.hpp

```diff
@@ -40,4 +44,45 @@ std::optional<ov::Layout> getLayoutFromRTMap(const ov::RTMap& rtMap);
 
 Status validatePluginConfiguration(const plugin_config_t& pluginConfig, const std::string& targetDevice, const ov::Core& ieCore);
 
+// Logging
+// #1 model/global plugin  CompiledMode:DUMMY / Global OpenVINO plugin:CPU
+// #2 version/_
+// #3 target_devide/_
+// {} {}
+template <typename PropertyExtractor>
+static void logOVPluginConfig(PropertyExtractor&& propertyExtractor, const std::string& loggingAuthor, const std::string& loggingDetails) {
+    SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Logging {}; {}plugin configuration", loggingAuthor, loggingDetails);
+    auto key = std::string("SUPPORTED_PROPERTIES");  // ov::supported_properties;
+    std::vector<ov::PropertyName> supportedConfigKeys;
+    try {
+        ov::Any value = propertyExtractor(key);
+        supportedConfigKeys = value.as<std::vector<ov::PropertyName>>();
+    } catch (std::exception& e) {
+        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Exception thrown from OpenVINO when requesting {};{} config key: {}; Error: {}", loggingAuthor, loggingDetails, key, e.what());
```

Comment:
```suggestion
        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Exception thrown from OpenVINO when requesting {};{}config key: {}; Error: {}", loggingAuthor, loggingDetails, key, e.what());
```

Reply:
I changed format in modelinstance.cpp now when triggered by requesting non existing key:
[2024-01-03 10:24:39.717][402856][modelmanager][debug][ov_utils.hpp:77] Exception thrown from OpenVINO when requesting OpenVINO Core plugin: GNA; config key: INFERENCE_PRECISION_HINT; Error: Exception from src/inference/src/core.cpp:228:
[ GENERAL_ERROR ] [openvino_intel_gna_plugin] in function GetMetric: Unsupported parameters for GetMetric: ABECADLO

[2024-01-03 10:24:39.734][402856][modelmanager][debug][ov_utils.hpp:77] Exception thrown from OpenVINO when requesting compiled model: dummy; version: 1; target device: CPU; config key: OPTIMAL_NUMBER_OF_INFER_REQUESTS; Error: Exception from src/inference/src/compiled_m
odel.cpp:119:
Exception from src/plugins/intel_cpu/src/compiled_model.cpp:193:
Unsupported property: ABECADLO